### DISCUSSION TITLE
Ledger Nano S shows 0 BTC and ETH balances, no trx history either

### DISCUSSION THREAD

**Discussion by user11992** [LedgerHQ/ledger-live-desktop#3989] [Q&A] [answered]
After updating my Ledger Nano S for the first time in 3+ years my balances and transaction history for BTC, ETH, and Dash show 0 and no transactions. They are completely gone. The only balance that shows is LTC.

The process I used to update my wallet are as follows:

I first deleted all apps and updated the firmware from 1.3.1 to 1.6 to 2.0.

I then reinstalled all apps and accounts to no avail.

I tried restoring my wallet using the seed phrase to no avail. (I did not create a second seed phrase, i restored from the original seed)

I also tried adding all the address types for BTC in Ledger Live and all show a 0 balance - Legacy, Segwit, Native Segwit.

And lastly, I tried using the Ledger Live experimental feature "extended address search" which is designed to find erroneous derivation paths and link them to a regular address. This did not work either.

I’m very concerned about how to retrieve my cryptos, these were not stolen but it sure feels that way. Ledger must have a solution for this but I cannot find it anywhere. I contacted support, left messages on social, still waiting for a reply.

I was quoted a very very very high amount of money for someone to try recovering with no guarantee of success.

Is it possible that my Ledger is corrupt and a new ledger restored with my original seed phrase could recover my "lost" coins? What am I missing?

Thanks.

**Comment by gre** [comment, marked as answer]
Github is not the place for this. Please contact official tech support which will investigate with you.
https://support.ledger.com
Thanks
